server/util_test.go

```diff
@@ -115,8 +116,9 @@ func TestInterceptConfigsPreRunHandlerReadsConfigToml(t *testing.T) {
 		t.Fatalf("Failed closing config.toml: %v", err)
 	}
 
-	cmd := server.StartCmd(nil, "/foobar")
-	if err := cmd.Flags().Set(flags.FlagHome, tempDir); err != nil {
+	cmd := server.StartCmd(nil)
+	cmd.PersistentFlags().String(flags.FlagHome, "/foober", "")
```

Comment:
Just for parity, the name of the command has changed from "/foobar" to "/foober" now. Kindly keep them consistent.